src/components/OrderConfirmation/OrderConfirmation.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { OrderItem } from '..';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+export class OrderConfirmation extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleClose = this.handleClose.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.addToOrder = this.addToOrder.bind(this);
+  }
+
+  handleClose() {
```

Comment:
If you used arrow functions to declare these then you could get rid of your `constructor` completely as you wouldn't have to `.bind(this)`

```js
handleClose = () => { // stuff }
```